### ISSUE TITLE
Tab width does not allow for entire label to be displayed

### ISSUE THREAD

**Issue by matthewgallo** [carbon-design-system/carbon#3579]
## Tab width too small for longer labels
The set width `10rem`s of the tab component restricts the label from being displayed entirely.

## Environment

> Browser
Chrome

## Detailed description

> What version of the Carbon Design System are you using?
10.3

> What did you expect to happen?
I'd like to propose a design change that enables a way to show the entire label, this issue is seen more frequently when our app is translated to German.

> What happened instead?
Tab label is truncated

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Use "transaction overview" in German, inside of a tab
2. See how label is truncated.

> Please create a reduced test case in CodeSandbox
> *https://codesandbox.io/s/codesandbox-hrx9r*
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
![Screen Shot 2019-07-30 at 10 01 40 AM](https://user-images.githubusercontent.com/10215203/62135902-0d15d180-b2b1-11e9-95c7-c0b9a16c8bcd.png)

![image (26)](https://user-images.githubusercontent.com/10215203/62135848-f3748a00-b2b0-11e9-8e23-1349886191b2.png)

- Notes

**Comment by asudoh** [comment]
Hi 👋 for long words considering different languages, you can increase the tab width via CSS.

**Comment by abbeyhrt** [comment]
Closing this since the problem was addressed! In this case, we would refer to what @asudoh said as the solution to the tab width. If that's not the case, feel free to open the issue back up!

**Comment by BrunoLongo-QA** [comment]
How can I change the width?

I tried 
`.bx--tabs__nav-label {
  width: 500px;
}`

`<Tabs {...props.tabs}>
            <Tab
              {...props.tab}
              id="Tab-Automation"
              label="Automation Request"
              className="bx--tabs__nav-label"
            >`

But the end result was something like this:
![image](https://user-images.githubusercontent.com/60791076/87203635-6f58be00-c2d9-11ea-91fc-cb48b508fa87.png)

**Comment by BrunoLongo-QA** [comment]
@asudoh @abbeyhrt can you help me?

**Comment by emyarod** [comment]
you'll probably want something like this https://github.com/carbon-design-system/carbon/issues/6898#issuecomment-697702219
